enterprise/query-logging/src/test/java/org/neo4j/kernel/impl/query/QueryLoggerTest.java

```diff
@@ -69,6 +69,7 @@
     public final FakeHeapAllocation heapAllocation = new FakeHeapAllocation();
     private long pageHits;
     private long pageFaults;
+    private long threshold = 10; //ms
```

Comment:
please rename variable instead to highlight that it's `ms` we are talking about here